src/frontend/packages/lib_common/src/queries/actionOne.tsx

```diff
@@ -1,13 +1,23 @@
 import { useJwt } from 'lib-components';
```

Comment:
This must not be moved in `lib-common` but rather in `lib-components`.
As discussed yesterday, by doing so, you are creating a circular dependency between `lib-common` and `lib-component` preventing the project to build unles allready build.
(`lib-common` now needs `lib-components` to build because you import `useJwt`, and `lib-components` need `lib-common` to build to import types as `Maybe`).

Reply:
Good catch !

Reply:
Ok, I had `lib_common` in my notes it's why.  
But if I understand correctly, when all will be in `lib_components` what about `import { Maybe } from 'lib-common';`  in `fetchList.tsx`? Is it not circular that as well?

Reply:
Since the file importing `Maybe` from `lib-components` is also in `lib-components`, you can just import it with a relative path.
Note that I already fix all this in fixup commits